tvb_documentation/demos/calcium_monitor_tutorial/tvb_model_reference/simulation_file/parameter/nuu_parameter_Mouse_512.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Feb 24 10:29:52 2021
+
+@author: nuuria
+"""
+
+import os
+
+class Parameter :
```

Comment:
This huge parameter dump here is not ok, especially since there are multiple such files with similar structure and same class names.